app/src/main/res/values/strings.xml

```diff
@@ -3,45 +3,45 @@
     <string name="app_name" translatable="false">Seedvault</string>
     <string name="backup">Backup</string>
     <string name="data_management_label">Seedvault Backup</string>
-    <string name="current_destination_string">Backup status and settings</string>
+    <string name="current_destination_string">Back up status and settings</string>
 
     <string name="restore_backup_button">Restore backup</string>
 
     <!-- Settings -->
     <string name="settings_category_apps">App backup</string>
-    <string name="settings_backup">Backup my apps</string>
+    <string name="settings_backup">Back up my apps</string>
     <string name="settings_backup_location">Backup location</string>
     <string name="settings_backup_location_none">None</string>
     <string name="settings_backup_location_internal">Internal storage</string>
     <string name="settings_backup_last_backup_never">Never</string>
-    <string name="settings_info">All backups are encrypted on your phone. To restore from backup you will need your 12-word recovery code.</string>
-    <string name="settings_auto_restore_title">Automatic restore</string>
```

Comment:
Automatic restoration, or automatic restoring would be possible options, this is however wrong.

Reply:
Restoration could work.